msticpy/datamodel/entities/alert.py

```diff
@@ -4,12 +4,15 @@
 # license information.
```

Comment:
There's a bit of an issue here (that I've helped to create). We should perhaps tidy this up and sweep up the SecurityAlert class at the same time.
In the V3 schema there is the SecurityAlert class and there is also an Alerts class - the latter was meant to be used to hold collections of alerts when you have an incident or an alert with related alerts (although seemed a bit of dumb kind of class to have IMHO - better to have just had edges).

We should probably consolidate into a single Alert V3 class that follows the V3 schema plus any Azure Sentinel extras.
And keep an Alerts glue entity. 
https://dev.azure.com/msazure/One/_git/Rome-Detection-AlertContracts?path=/src/AlertContracts/V3/Entities/Alerts.cs
vs.
https://dev.azure.com/msazure/One/_git/Rome-Detection-AlertContracts?path=/doc/AlertV3.md&_a=preview

Actually the former doesn't even appear in the docs so maybe we can just drop Alerts and consolidate the Alert one.
In the V3 schema SecurityAlert is not an entity but a separate thing. Not sure we should end up making it an entity.
Maybe we should put the Alert and Incident classes in a peer folder like soc or something.